lib/kafka/protocol/offset_fetch_request.rb

```diff
@@ -13,7 +13,7 @@ def api_key
       end
 
       def api_version
-        1
+        @topics.nil? ? 2 : 1
```

Comment:
Can you add a comment here documenting why this is necessary?

Reply:
Done